main.py

```diff
@@ -51,14 +52,13 @@ def reAward(fname):
 ##(/5) Task 3: CREATE code that will open a file and extract all the phone numbers 
 
 def rePhone(fname):
-  #open file
-  #loop through the file
-  #extract the specific phone numbers regex
-  #if the length of the extraction is not empty
-  #print the phone number
-  pass
-
+  fhand = open(fname, 'r')
+  for line in fhand:
+    extr = re.findall('902.*', line)
```

Comment:
Look at the phone number pattern.   ###-###-####
`extr = re.findall('\d{3}-\d{3}-\d{4}', line)`